### PR TITLE
3Dconnexion.3Dconnexion3DxWare10 version 10.8.8.3488

### PR DESCRIPTION
**Pull request by TheBossMagnus**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/87648)

### CODE REVIEW

File: manifests/3/3Dconnexion/3Dconnexion3DxWare10/10.8.8.3488/3Dconnexion.3Dconnexion3DxWare10.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: 3Dconnexion.3Dconnexion3DxWare10
```
Comment (Trenly) [inline manifests/3/3Dconnexion/3Dconnexion3DxWare10/10.8.8.3488/3Dconnexion.3Dconnexion3DxWare10.installer.yaml:4]:
```suggestion
PackageIdentifier: 3Dconnexion.3DxWare.10
```

File: manifests/3/3Dconnexion/3Dconnexion3DxWare10/10.8.8.3488/3Dconnexion.3Dconnexion3DxWare10.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: 3Dconnexion.3Dconnexion3DxWare10
```
Comment (Trenly) [inline manifests/3/3Dconnexion/3Dconnexion3DxWare10/10.8.8.3488/3Dconnexion.3Dconnexion3DxWare10.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: 3Dconnexion.3DxWare.10
```

File: manifests/3/3Dconnexion/3Dconnexion3DxWare10/10.8.8.3488/3Dconnexion.3Dconnexion3DxWare10.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: 3Dconnexion.3Dconnexion3DxWare10
+PackageVersion: 10.8.8.3488
+PackageLocale: en-US
+Publisher: 3Dconnexion
+PackageName: 3Dconnexion 3DxWare 10
+License: 1edsfsdf
```
Comment (Trenly) [inline manifests/3/3Dconnexion/3Dconnexion3DxWare10/10.8.8.3488/3Dconnexion.3Dconnexion3DxWare10.locale.en-US.yaml:9]: What kind of license is this?

File: manifests/3/3Dconnexion/3Dconnexion3DxWare10/10.8.8.3488/3Dconnexion.3Dconnexion3DxWare10.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: 3Dconnexion.3Dconnexion3DxWare10
+PackageVersion: 10.8.8.3488
+PackageLocale: en-US
+Publisher: 3Dconnexion
+PackageName: 3Dconnexion 3DxWare 10
+License: 1edsfsdf
+Copyright: Copyright © 2004- 3Dconnexion. All rights reserved.
+ShortDescription: 3Dconnexion 3DxWare 10 (64-bit)
```
Comment (Trenly) [inline manifests/3/3Dconnexion/3Dconnexion3DxWare10/10.8.8.3488/3Dconnexion.3Dconnexion3DxWare10.locale.en-US.yaml:11]: Please provide a better description of what 3DxWare does

File: manifests/3/3Dconnexion/3Dconnexion3DxWare10/10.8.8.3488/3Dconnexion.3Dconnexion3DxWare10.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: 3Dconnexion.3Dconnexion3DxWare10
```
Comment (Trenly) [inline manifests/3/3Dconnexion/3Dconnexion3DxWare10/10.8.8.3488/3Dconnexion.3Dconnexion3DxWare10.yaml:4]:
```suggestion
PackageIdentifier: 3Dconnexion.3DxWare.10
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by TheBossMagnus** [comment]
I accidentally submit this pr, anyway there is a more recent version available. Closing

**Pull request closed without merging**